Client/src/Components/TabPanels/Account/TeamPanel.jsx

```diff
@@ -172,6 +172,10 @@ const TeamPanel = () => {
   };
 
   const handleInviteMember = async () => {
+    if (!toInvite.email) {
+      setErrors((prev) => ({ ...prev, email: "Email is required." }));
+      return;
+    }
```

Comment:
This works, however I think from a UX standpoint it makes more sense to have the `send invite` button disabled until the user actually has inputted something.  Otherwise the user can potentially have thier time wasted by clicking submit with a form that is guaranteed to fail.

How about tracking the state of the input and errors?

```
  const [isDisabled, setIsDisabled] = useState(true);

  useEffect(() => {
    setIsDisabled(Object.keys(errors).length !== 0 || toInvite.email === "");
  }, [errors, toInvite.email]);
```

and setting the disabled state of the loading button based on that?

```
<LoadingButton
  variant="contained"
  color="primary"
  onClick={handleInviteMember}
  loading={isSendingInvite}
  disabled={isDisabled}
>
  Send invite
</LoadingButton>
```